net/openfortivpn/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        adrienverge openfortivpn 1.6.0 v
+github.tarball_from tags
+
+categories          net
+platforms           darwin
+license             GPL-3
+
+maintainers         @millerdev
```

Comment:
You need to add your email address to this, in the form of domain:user, as in

`maintainers {foo.com:bar @millerdev}`

Unless you have a good reason to do otherwise, you should also probably add `openmaintainer`.